composer.json

```diff
@@ -30,6 +30,7 @@
     "require": {
         "php": "^5.5.9|~7.0",
         "psr/log": "^1.0",
+        "symfony/symfony": "2.8.*",
```

Comment:
Ah yes of course, that changed as I was testing on early Symfony versions. I'll sort that now.